sos/plugins/kubernetes.py

```diff
@@ -22,33 +23,101 @@ class kubernetes(Plugin, RedHatPlugin):
     """Kubernetes plugin
     """
 
-    option_list = [("podslog", "capture logs for pods", 'slow', False)]
+    # Red Hat Atomic Platform and OpenShift Enterprise use the
+    # atomic-openshift-master package to provide kubernetes
+    packages = ('kubernetes', 'atomic-openshift-master')
```

Comment:
Side note: the [OpenShift Enterprise installation guide](https://access.redhat.com/documentation/en/openshift-enterprise/3.1/installation-and-configuration/chapter-2-installing#rpm-vs-containerized) has more details of the "containerized install" option for OpenShift Enterprise.

I know that this is a generic Kubernetes plugin and that the pointer above is just one use case for it. Maybe we can add a separate RedHatPlugin class and have that include `files` to cover that scenario? (while waiting for a proper 'extrospection' implementation)